core/server/worker/src/main/java/alluxio/worker/netty/DataServerReadHandler.java

```diff
@@ -165,7 +166,9 @@
 
   @Override
   public void channelUnregistered(ChannelHandlerContext ctx) {
-    setError(ctx.channel(), new Error(null, false, Protocol.Status.Code.INTERNAL));
+    // TODO(andrew): talk to peis about whether this is the right exception type
```

Comment:
@peisun1115 Does this mean an internal invariant has been broken, or is a different status code more appropriate? https://github.com/grpc/grpc-go/blob/master/codes/codes.go